libuavcan/include/uavcan/driver/can.hpp

```diff
@@ -85,6 +85,16 @@ struct UAVCAN_EXPORT CanFilterConfig
 {
     uint32_t id;
     uint32_t mask;
+
+    bool operator == (const CanFilterConfig& rhs) const
```

Comment:
Did uncrustify put spaces around `==`? If so, sorry, I'll try to teach it otherwise. If not, please remove the spaces ().